src/components/Nav.jsx

```diff
@@ -9,6 +9,7 @@ import { Link, withRouter } from 'react-router'
 
 const Nav = ({ t, router }) => {
   let isBrowsingFiles = router.location.pathname.match(/^\/files/) !== null
+  let isBrowsingTrash = router.location.pathname.match(/^\/trash/) !== null
```

Comment:
I feel like there should be a better, more global way to do this. As you can see later in this PR, I've even added a reducer that does it and used it in a couple of places. However, we haven't one for `isBrowsingFiles `, so maybe we should add that too? Then again, the router is a relatively good way of finding out what the user is viewing.